frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -201,9 +210,27 @@ class ManageAccounts extends Component<Props, State> {
         });
     }
 
+    private updateAccountName = (event: Event) => {
+        event.preventDefault();
+        const { editAccountCode, editAccountNewName } = this.state;
+
+        backendAPI.renameAccount(editAccountCode!, editAccountNewName!)
+            .then(result => {
+                if (!result.success) {
+                    this.setState({ editErrorMessage: result.errorMessage });
```

Comment:
it looks nice but code-wise it might be overkill - it should be impossible to get an error (if you prevent sending an empty name). for such errors, we often simply do alertUser(...) to show the message, in case an unexpected error comes out.

no need to change, just a thought.